contract-examples/test/contract_deployer_allow_list.ts

```diff
@@ -85,31 +80,6 @@ describe("ExampleDeployerList", function () {
     expect.fail("should have errored")
   });
 
-  it("should not allow deployer to enable itself", async function () {
-    try {
-      await contract.connect(deployer).addDeployer(deployer.address);
-    }
-    catch (err) {
-      return
-    }
-    expect.fail("should have errored")
-  });
```

Comment:
`addDeployer` isn't a contract function, so this test really isn't testing anything. The allowlist logic is already tested.

Reply:
https://hardhat.org/hardhat-runner/docs/advanced/create-task Read more about tasks here. Seems to be used by hardhat as a cli command. These should be removed 👍